crates/bevy_gltf/src/loader/mod.rs

```diff
@@ -191,6 +191,8 @@ pub struct GltfLoaderSettings {
     pub load_cameras: bool,
     /// If true, the loader will spawn lights for gltf light nodes.
     pub load_lights: bool,
+    /// If true, the loader will load animations. Requires the `bevy_animation` feature.
```

Comment:
```suggestion
    /// If true, the loader will load `AnimationClip` assets, and also add
    /// `AnimationTarget` and `AnimationPlayer` components to hierarchies
    /// affected by animation. Requires the `bevy_animation` feature.
```

Maybe a bit pedantic, but being precise here could help some users. Say someone disables animations in a primary skinned mesh glTF but also loads animations from a secondary glTF - their skinned mesh will not have the `AnimationTarget` and `AnimationPlayer` components required to play the animations.

I'm working on a PR that changes some of these internals, so the comment might get simpler in future.